### DISCUSSION TITLE
Street Fighter 6

### DISCUSSION THREAD

**Discussion by Whatsmyename** [praydog/REFramework#811] [General Questions]
Will this mod work with street fighter 6?

**Comment by Yudoken** [comment]
Apparently the [RE4 version works](https://steamcommunity.com/app/1364780/discussions/0/3830917450435979150/) but is kinda glitchy, I would highly appreciate if REF could be released for SF 6 too. The anti aliasing and picture clarity is pretty bad and with DLSS it would be a major upgrade especially in world tour mode. It's quite taxing and does not quite reach the fps I aim for with my RTX 3080 @4k with almost maxed settings. The fps fluctuates quite hard in the overworld and increasing the internal resolution or adding regular anti aliasing is a must with the amount of aliasing even in 4k which additionally performance away.

For enabling HDR I use special k since it's the only way possible and the game looks great with it.
What I would love to see are options to increase the lod distance, the game aggressively limits it and animations look awefull in the distance with the heavily limited fps.

Obviously it's something far from it but maybe, just maybe the games fps can also be unlocked in regular games like with the Tekken 7 overlay mod. I know it didn't work out apparently but It's a huge upgrade and playing games again at locked 60 is disappointing.

It's a lot to ask for a newly released game and I appreciate your efforts. This mod is amazing and I would love to see it working with SF 6 with the DLSS support.
